### ISSUE TITLE
Remove patch of chrome/test/views/accessibility_checker.cc

### ISSUE THREAD

**Issue by darkdh** [brave/brave-browser#748]
Currently simple test case like
```
1. NavigateToURLUntilLoadStop(URL a)
2. ExecuteScriptAndExtractString
3. NavigateToURLUntilLoadStop(URL b)
4. ExecuteScriptAndExtractString
``` 
failed with
```
../../chrome/test/views/accessibility_checker.cc:120: Failure
Failed
The following view violates DoesViewHaveAccessibilityErrors() when its widget becomes hidden:
browser/ui/views/frame/BrowserRootView > ui/views/window/NonClientView > BrowserView > TopContainerView > TabStrip > Tab > AlertIndicatorButton (id 0)
- Focusable View has no accessible name or placeholder, and the name attribute does not use kAttributeExplicitlyEmpty.

Note: for a more useful error message that includes a stack of how this view was constructed, use git cl patch 963284. Please leave a note on that CL if you find it useful.
Stack trace:
0   libbase.dylib                       0x000000011e2ac16c base::debug::StackTrace::StackTrace() + 28
1   brave_browser_tests                 0x0000000105d56c0a StackTraceGetter::CurrentStackTrace(int, int) + 90
2   brave_browser_tests                 0x0000000105d7aae7 testing::internal::UnitTestImpl::CurrentOsStackTraceExceptTop(int) + 71
3   brave_browser_tests                 0x0000000105d79e5b testing::internal::AssertHelper::operator=(testing::Message const&) const + 139
4   brave_browser_tests                 0x000000010daf08f3 AddFailureOnWidgetAccessibilityError(views::Widget*) + 403
5   brave_browser_tests                 0x000000010daf0d90 AccessibilityChecker::OnWidgetVisibilityChanged(views::Widget*, bool) + 32
6   libviews.dylib                      0x000000013b60ac1a views::Widget::OnNativeWidgetVisibilityChanged(bool) + 218
7   libviews.dylib                      0x000000013b46518a views::BridgedNativeWidget::OnVisibilityChanged() + 586
8   libviews.dylib                      0x000000013b47cef8 -[ViewsNSWindowDelegate onWindowOrderChanged:] + 40
9   libviews.dylib                      0x000000013b47b86b -[NativeWidgetMacNSWindow orderWindow:relativeTo:] + 123
10  libviews.dylib                      0x000000013b461cd0 views::BridgedNativeWidget::SetVisibilityState(views::BridgedNativeWidget::WindowVisibilityState) + 352
11  libviews.dylib                      0x000000013b5f5f2c views::NativeWidgetMac::Hide() + 108
12  libviews.dylib                      0x000000013b6093bd views::Widget::Hide() + 29
13  brave_browser_tests                 0x0000000106f05000 BrowserView::CanClose() + 256
14  libviews.dylib                      0x000000013b629080 views::NonClientView::CanClose() + 32
15  libviews.dylib                      0x000000013b608902 views::Widget::Close() + 98
16  brave_browser_tests                 0x0000000106ef7ebe BrowserView::Close() + 46
17  brave_browser_tests                 0x000000010c6e15ec BrowserCloseManager::CloseBrowsers() + 1996
18  brave_browser_tests                 0x000000010c6e1bf5 BrowserCloseManager::CheckForDownloadsInProgress() + 21
19  brave_browser_tests                 0x000000010c6e1a16 BrowserCloseManager::TryToCloseBrowsers() + 374
20  brave_browser_tests                 0x000000010c6e0e13 BrowserCloseManager::StartClosingBrowsers() + 67
21  brave_browser_tests                 0x000000010bf3e387 chrome::CloseAllBrowsers() + 135
22  brave_browser_tests                 0x000000010bf3e2f3 chrome::CloseAllBrowsersAndQuit() + 19
23  brave_browser_tests                 0x000000010baab514 -[AppController tryToTerminateApplication:] + 660
24  brave_browser_tests                 0x000000010bc2de1a -[BrowserCrApplication terminate:] + 74
25  brave_browser_tests                 0x000000010bc2d572 chrome_browser_application_mac::Terminate() + 34
26  brave_browser_tests                 0x000000010bb2d0b1 BrowserProcessPlatformPart::AttemptExit() + 17
27  brave_browser_tests                 0x000000010bf3e2b4 chrome::AttemptExitInternal(bool) + 148
28  brave_browser_tests                 0x000000010bf3e9f5 chrome::AttemptExit() + 37
29  brave_browser_tests                 0x0000000104920aaa void base::internal::FunctorTraits<void (*)(), void>::Invoke<void (* const&)()>(void (* const&&&)()) + 26
30  brave_browser_tests                 0x0000000104920a7d void base::internal::InvokeHelper<false, void>::MakeItSo<void (* const&)()>(void (* const&&&)()) + 29
31  brave_browser_tests                 0x0000000104920a51 void base::internal::Invoker<base::internal::BindState<void (*)()>, void ()>::RunImpl<void (* const&)(), std::__1::tuple<> const&>(void (* const&&&)(), std::__1::tuple<> const&&&, std::__1::integer_sequence<unsigned long>) + 33
32  brave_browser_tests                 0x00000001049209dc base::internal::Invoker<base::internal::BindState<void (*)()>, void ()>::Run(base::internal::BindStateBase*) + 44
33  libbase.dylib                       0x000000011e25261c base::OnceCallback<void ()>::Run() && + 92
34  libbase.dylib                       0x000000011e2add29 base::debug::TaskAnnotator::RunTask(char const*, base::PendingTask*) + 1033
35  libbase.dylib                       0x000000011e38149d base::MessageLoop::RunTask(base::PendingTask*) + 877
36  libbase.dylib                       0x000000011e381a89 base::MessageLoop::DeferOrRunPendingTask(base::PendingTask) + 89
37  libbase.dylib                       0x000000011e381f2a base::MessageLoop::DoWork() + 570
38  libbase.dylib                       0x000000011e38f2c2 base::MessagePumpCFRunLoopBase::RunWork() + 98
39  libbase.dylib                       0x000000011e38f24c ___ZN4base24MessagePumpCFRunLoopBase13RunWorkSourceEPv_block_invoke + 28
40  libbase.dylib                       0x000000011e32ce0a base::mac::CallWithEHFrame(void () block_pointer) + 10
41  libbase.dylib                       0x000000011e38e125 base::MessagePumpCFRunLoopBase::RunWorkSource(void*) + 101
42  CoreFoundation                      0x00007fff30732a11 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
```
`accessibility_checker.cc is` is introduced in Chromium 68(https://chromium.googlesource.com/chromium/src/+/373e1cddeafe7b6ec8ebbef802bda4ab9f69b2c2), we had to use work around to bypass the error and this can only be reproduced by running the failed cases along with all browser tests

failed check node is `id=-1 button FOCUSABLE (294, 61)-(16, 16) action=press name= class_name=AlertIndicatorButton actions=`

Remove the patch once upstream fixes the bug

**Comment by bsclifton** [comment]
Closing - this patch has been removed
